--- a/src/MainWindow.cpp
+++ b/src/MainWindow.cpp
@@ -307,7 +307,11 @@
     FileInfo * sel = selectedItems.first();
     bool oneDirSelected = ( selectedItems.size() == 1 ) && sel && sel->isDir();
 
+#ifdef ENABLE_TRASH
     _ui->actionMoveToTrash->setEnabled( sel && ! sel->isDotEntry() );
+#else
+    _ui->actionMoveToTrash->setEnabled( false );
+#endif
     _ui->actionRefreshSelected->setEnabled( oneDirSelected && ! sel->isExcluded() && ! sel->isMountPoint() );
     _ui->actionContinueReadingAtMountPoint->setEnabled( oneDirSelected && sel->isMountPoint() );
     _ui->actionReadExcludedDirectory->setEnabled      ( oneDirSelected && sel->isExcluded()   );
