--- a/src/MainWindow.cpp
+++ b/src/MainWindow.cpp
@@ -307,7 +307,11 @@
     bool pseudoDirSelected = selectedItems.containsPseudoDir();
     bool pkgSelected	   = selectedItems.containsPkg();
 
+#ifdef ENABLE_TRASH
     _ui->actionMoveToTrash->setEnabled( sel && ! pseudoDirSelected && ! pkgSelected && ! reading );
+#else
+    _ui->actionMoveToTrash->setEnabled( false );
+#endif
     _ui->actionRefreshSelected->setEnabled( selSize == 1 && ! sel->isExcluded() && ! sel->isMountPoint() && ! pkgView );
     _ui->actionContinueReadingAtMountPoint->setEnabled( oneDirSelected && sel->isMountPoint() );
     _ui->actionReadExcludedDirectory->setEnabled      ( oneDirSelected && sel->isExcluded()   );
